src/References.php

```diff
@@ -69,6 +69,12 @@ public function mergeWith(References $references): References
 
         foreach ($references as $filename => $lines) {
             if (empty($lines)) {
+                //Check the $filename type
+                if (gettype($filename) == "integer") {
+                    //Set string
+                    $filename = strval($filename);   
+                }
+
                 $merged->add($filename);
```

Comment:
This only solves the problem inside the `if`.
Maybe a better solution would be:

```php
foreach ($references as $filename => $lines) {
    $filename = (string) $filename;

    if (empty($lines)) {
```

Reply:
Ok, I've tried with your suggestion and works!